### PR TITLE
UI for organization select when multiple organizations found

### PR DESCRIPTION
**Pull request by sadilchamishka**
### Purpose
In B2B organization management, when a user try to login to SAAS apps via the enterprise login, user is prompted to provide the organization. But there can be organization with same name subscribed for the same SAAS app. In that case this UI will be prompted for the user with organization name and its description to select the intended organization easily.

Screenshot 2022-07-06 at 14 22 14

### Related Issues
https://github.com/wso2/product-is/issues/14099

### CODE REVIEW

File: apps/authentication-portal/src/main/webapp/select_org.jsp [hunk 1]
```diff
@@ -0,0 +1,158 @@
+<%--
+  ~ Copyright (c) 2022, WSO2 Inc. (http://www.wso2.com).
```
Comment (madurangasiriwardena) [inline apps/authentication-portal/src/main/webapp/select_org.jsp:2]:
```suggestion
  ~ Copyright (c) 2022, WSO2 LLC. (http://www.wso2.com).
```

Comment (sadilchamishka) [reply apps/authentication-portal/src/main/webapp/select_org.jsp]: Fixed by - https://github.com/wso2/identity-apps/pull/3135/commits/7336cd6ec21d68b1afefe9be320ce89555f49e17

**Review by madurangasiriwardena — APPROVED** [review]

File: apps/authentication-portal/src/main/webapp/select_org.jsp [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+
+<%
+
+    String idp = request.getParameter("idp");
+    String authenticator = request.getParameter("authenticator");
+    String sessionDataKey = request.getParameter(Constants.SESSION_DATA_KEY);
+    int orgCount = Integer.parseInt(request.getParameter("orgCount"));
+
+    String errorMessage = AuthenticationEndpointUtil.i18n(resourceBundle, "error.retry");
+    String authenticationFailed = "false";
```
Comment (chamathns) [inline apps/authentication-portal/src/main/webapp/select_org.jsp:35]: Why are we using a string here instead of boolean?

Comment (sadilchamishka) [reply apps/authentication-portal/src/main/webapp/select_org.jsp:35]: Yes, it should be boolean. Addressed by this PR https://github.com/wso2/identity-apps/pull/3523. 

In this PR I had a requirement to send list of organization ids, names and descriptions to the JSP file. What I have done is grouped individually encoded id,name,description by commas and concat these sets using `:` and send as query from to the jsp file. Is there any recommended way to achieve that type of requirement.

**Pull request merged**